Services/Interfaces/web_interface/models/configuration.py

```diff
@@ -537,6 +538,31 @@ def get_symbol_list(exchanges):
 
     return list(set(result))
 
+def get_timeframes_list(exchanges):
+    config = {commons_constants.CONFIG_TIME_FRAME: [], commons_constants.CONFIG_CRYPTO_CURRENCIES: {}}
+    timeframes_list = []
+    allowed_timeframes = set(tf.value for tf in commons_enums.TimeFrames)
+    for exchange in exchanges:
+        if exchange not in exchange_symbol_fetch_blacklist:
+            try:
+                exchange_manager = interfaces_util.run_in_bot_async_executor(
```

Comment:
Why instantiating a new exchange manager instead of doing the same thing as `get_symbol_list` ?

Reply:
if it's not a ccxt exchange or if ccxt have made some change on this

Reply:
That's right. I think it's a bit overkill to create an exchange mamanger instance for that but we don't have any API to do that for now

Reply:
so i fetch it with ccxt for now ?

Reply:
I think so, what do you think @GuillaumeDSM ?

Reply:
I'm adding an api on octobot-trading do to this.
I think it would also be good to store these results into a global buffer (timeframes by exchanges) in order not to re-querry the same thing all the time